website/docs/docs/build/unit-tests.md

```diff
@@ -20,6 +20,7 @@ Now, we are introducing a new type of test to dbt - unit tests. In software prog
 - You must specify all fields in a BigQuery STRUCT in a unit test. You cannot use only a subset of fields in a STRUCT.
 - If your model has multiple versions, by default the unit test will run on *all* versions of your model. Read [unit testing versioned models](#unit-testing-versioned-models) for more information.
 - Unit tests must be defined in a YML file in your `models/` directory.
+- Table names must be [aliased](/docs/build/custom-aliases) to unit test `join` logic.
```

Comment:
Minor suggestion so it doesn't read like they need to be "aliased to the unit test join logic"
```suggestion
- Table names must be [aliased](/docs/build/custom-aliases) in order to unit test `join` logic.
```